### DISCUSSION TITLE
Can ozone replace minio in Milvus? If so, how can the connection be configured?

### DISCUSSION THREAD

**Discussion by a28124791** [apache/ozone#7158] [FAQ] [answered]
Can ozone replace minio in Milvus? If so, how can the connection be configured?
ozone 是否可以替代Milvus中的minio？可以的化需要怎么配置连接？

**Comment by kerneltime** [comment, marked as answer]
> Can ozone replace minio in Milvus? If so, how can the connection be configured? ozone 是否可以替代Milvus中的minio？可以的化需要怎么配置连接？

Ozone like MinIO is a S3 compatible storage engine which also supports HDFS. You would need to change the URL to the S3 Gateway address and replace the new credentials generated by Ozone.

**Comment by umamaheswararao** [comment]
I think milvus has external s3 configurations. I never tried personally, but that should be the best way to connect to Ozone.
https://milvus.io/docs/v2.0.x/operator.md#Configure-object-storage
